chart/values.yaml

```
@@ -1 +1,9 @@
 extraPorts: []
+
+coreDnsOverrides: |
+  rewrite {
+    name regex (.*admin\.uds\.dev) admin-ingressgateway.istio-admin-gateway.svc.cluster.local answer auto
```

Is this correct? It might work as is, but probably is more accurate as:
```suggestion
    name regex (.*\.admin\.uds\.dev) admin-ingressgateway.istio-admin-gateway.svc.cluster.local answer auto
```

But might be missing something.

---

Where this could cause an issue in theory - an address like `myadmin.uds.dev` is hosted on the tenant gateway but would make this first regex I think, incorrectly routing its request to the admin gateway.

---

Implemented! That doesn't cause and issue and its something I've done before already, just forgot to put it in here.